common-content/en/module/tools/comparing-javascript-and-python/index.md

```diff
@@ -0,0 +1,68 @@
++++
+title = "Comparing JavaScript and Python"
+headless = true
+time = 40
+facilitation = false
+emoji= "📖"
+objectives = [
+    "Identify and explain equivalences between JavaScript and Python",
+    "Identify and explain differences between JavaScript and Python",
+    "Distinguish between essential and accidental complexity"
+]
++++
+
+JavaScript and Python are quite similar languages in a lot of ways.
+
+Most of the differences between them are quite cosmetic. e.g.
+* Some functions and operators have different names. But often there are functions/operators which do exactly the same thing.
+* JavaScript uses `{}` around blocks of code and we optionally _choose_ to indent code, whereas Python uses `:` and _required_ indents.
+* In JavaScript we choose to name variables in `camelCase`, whereas in Python we choose to name variables in `snake_case` (but in both langues we _could_ do either).
```

Comment:
```suggestion
* In JavaScript we choose to name variables in `camelCase`, whereas in Python we choose to name variables in `snake_case` . In both langues we _could_ do either; this is called a "convention". 
```

I would tooltip on convention because you're going to talk a bunch about convention.